src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -331,3 +331,67 @@ def load_arguments(self, _):
     with self.argument_context('monitor activity-log alert scope remove') as c:
         c.argument('scopes', options_list=['--scope', '-s'], nargs='+')
     # endregion
+
+    # region LogAlerts
+    with self.argument_context('monitor log-alert') as c:
+        c.argument('rule_name', options_list=['--name', '-n'], id_part='name', help="Name of the log alert rule.")
+
+    with self.argument_context('monitor log-alert create') as c:
+        c.argument('location', help="Resource location.")
+        c.argument('disable', action='store_true', help="Disable the log alert rule after it is created.")
+        c.argument('description', help="The description of the log alert rule.")
+        c.argument('tags', help="Space-separated tags in 'key[=value]' format.")
```

Comment:
there is `tags_type`  in `from azure.cli.core.commands.parameters`. You can just use it.